jekyll-remote-theme.gemspec

```diff
@@ -18,13 +18,13 @@ Gem::Specification.new do |s|
   s.add_dependency "addressable", "~> 2.0"
   s.add_dependency "jekyll", ">= 3.5", "< 5.0"
   s.add_dependency "rubyzip", ">= 1.2.1", "< 3.0"
+  s.add_dependency "jekyll-sass-converter", "~> 2.0"
```

Comment:
You may want to relax the constraint to allow using `jekyll-sass-converter-1.x` if you intend to support Jekyll 3.x as well because `Theme#configure_sass` calls on `Sass` module which is not loaded by the sass-converter v2.0